components/common/party/Cusor.vue

```diff
@@ -10,7 +10,7 @@
       :id="`cursor-${connection.id}`"
       icon="arrow-pointer"
       pack="fas"
-      :class="cursorDetails.color" />
+      :class="color" />
```

Comment:
how about this directly?
```suggestion
      :class="cursorDetails.color()" />
```

Reply:
i see, thank you